### ISSUE TITLE
Improvement: dropdown="false" atribute

### ISSUE THREAD

**Issue by JesusSD** [vaadin/vaadin-combo-box#162]
I need a dropdown="false" atribute for not open dropdown list and hide the dropdown icon.

**Comment by Saulis** [comment]
Hi!

We are adding a `read-only` attribute in #141 which is due to release in `1.0.0-alpha6`. It prevents the dropdown from opening but also prevents any input to be placed. Would this work for you?

You can try out the latest by installing the master version:

```bower install vaadin-combo-box#master```

**Comment by JesusSD** [comment]
I need that sometimes the drop-down list is not displayed but if they can enter values, not the "drop-dawn" icon is displayed.

For example, when you do not find items in the list.

Thank you.

**Comment by jounik** [comment]
If I understand you correctly, you’re looking for the "suggestion/autocomplete" feature, which only shows the matching dropdown items when the user is typing something in the field.

That feature is planned as a future improvement

**Comment by JesusSD** [comment]
No, for me it would be enough that the icon is not displayed when there were no elements in the list.

**Comment by jounik** [comment]
Okay, so if you do not provide an empty array for the `items` property, the dropdown button would be hidden?

That does sound quite reasonable to me, although it does make the component quite useless, unless you allow custom input values (which will be opt-in starting from 1.0.0-alpha6 #154).

**Comment by manolo** [comment]
@JesusSD sent a PR that should cover this use case, can you verify that it covers this issue ?
Thanks

**Comment by manolo** [comment]
So as discussed during the PR the final solution is a `has-items` attribute that is true when the element has items.
